templates/composer-project.sh.j2

```diff
@@ -0,0 +1 @@
+export PATH=$PATH:{{ composer_project_path }}/vendor/bin
```

Comment:
I think the composer project path should come before `$PATH` so that eg. a project specific drush takes precedence over the globally installed one. No?